hecuba_core/src/py_interface/UnitParser.cpp

```diff
@@ -299,4 +300,257 @@ PyObject *UuidParser::c_to_py(const void *payload) const {
     CassUuid uuid = {*((uint64_t *) it), *((uint64_t *) it + 1)};
     cass_uuid_string(uuid, final);
     return PyString_FromString(final);
+}
+
+TupleParser::TupleParser(const ColumnMeta &CM) : UnitParser(CM) {
+    this->col_meta = CM;
+}
+
+
+
+int16_t TupleParser::py_to_c(PyObject *obj, void *payload) const {
+    if (obj == Py_None) return -1;
```

Comment:
Check obj is a tuple with PyTuple_Check() or sth similar

Reply:
Also check :
PyTuple_Size(obj); == CM.pointer.size(), else throw ModuleException()